packages/react-icons/package.json

```
@@ -0,0 +1,39 @@
+{
+    "name": "@coveord/plasma-react-icons",
+    "version": "1.0.0",
+    "description": "Plasma iconography exposed as react components",
+    "main": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "scripts": {
+        "build": "node ./bin/index.js && tsc --project tsconfig.json"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/coveo/plasma.git",
+        "directory": "packages/react-icons"
+    },
+    "keywords": [
+        "plasma",
+        "icons",
+        "react"
+    ],
+    "author": "",
+    "license": "ISC",
+    "bugs": {
+        "url": "https://github.com/coveo/plasma/issues"
+    },
+    "homepage": "https://github.com/coveo/plasma/packages/react-icons#readme",
+    "files": [
+        "dist"
+    ],
+    "devDependencies": {
+        "@coveord/plasma-tokens": "workspace:*",
+        "@svgr/core": "^6.2.1",
```

Remove the carats, Renovate won't like it :see_no_evil: 
```suggestion
        "@svgr/core": "6.2.1",
```